OfekQuery.html

```diff
@@ -0,0 +1,74 @@
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="css/bootstrap.css">
+	<link rel="stylesheet" type="text/css" href="css/index.css">
+	<link rel="stylesheet" type="text/css" href="css/OfekQuery.css">
+
+</head>
+
+<body>
+
+<!-- Nav Bar -->
+<nav class="navbar navbar-default nav-color">
+	<div class="container-fluid">
+
+		<div class="navbar-header">
+			<img src="images/ofek.png" class="nav navbar-left logo-pic"/>
+			<a class="navbar-brand navbar nav-text logo-color" href="index.html">
+				OfekTwitter
+			</a>
+		</div>
+
+		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-3">
+			<ul class="nav navbar-nav navbar-right">
+				<li><a href="users.html" style="color: white">Users</a></li>
+				<li><a href="SignIn.html" style="color: white">Sign in</a></li>
+				<li><a href="SignUp.html" style="color: white">Sign up</a></li>
```

Comment:
You use the same style 3 times, why not add a class in the css?
It's not a good practice to use inline styling when not needed.